studio/__mocks__/hooks.js

```diff
@@ -8,6 +8,8 @@ export const useParams = jest.fn().mockImplementation(() => ({ ref: '123' }))
 
 export const useLocalStorage = jest.fn().mockImplementation(() => [[], jest.fn()])
 
+export const useIsFeatureEnabled = jest.fn().mockImplementation(() => [])
```

Comment:
Should we actually check the arg and return `true` if a string is passed in an `{}` if an array is passed in? Never returns a boolean does it?

Reply:
True – good catch – I've updated!